connector_acp/models/backend_acp.py

```diff
@@ -19,6 +19,11 @@ class BackendAcp(models.Model):
     status = fields.Selection((('unconfirmed', 'Unconfirmed'),
                                ('confirmed', 'Confirmed')), string="Status",
                               default='unconfirmed')
+    type_of_connection = fields.Selection(
+        [('usb', 'USB'),
+         ('wifi', 'WIFI')],
```

Comment:
Make this an empty list for now.
If that error, then uase a single option `[('nd', 'Not defined')]`